test/common/deployment_types.go

```diff
@@ -334,8 +407,53 @@ func TestDeploymentConfigExpectedReplicas(t TestingTB, ctx *TestingContext) {
 				)
 				continue
 			}
-
-			if deploymentConfig.Status.AvailableReplicas < product.ExpectedReplicas {
+			if rhmi.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
```

Comment:
Do we need this `if` block in general ? Seems we are just checking number of pods running vs expected number ? I don't there'll be much harm in having this run in RHMI e2e also unless I'm missing something ? :thinking:

Reply:
Not sure about that one. The reason why we decided to bother ourselves with checking values in pods in the first place was an introduction of Quota. Idea was to check not only that we set values in CRs but also that we are setting values that could be accepted by various products and could be propagated down to pods. 
For RHMI nothing gets changed so the default CR check should be sufficient. In theory, nothing holds us from checking CPU and memory in pods for RHMI (the code to do so is already here), but I think that if we decide to refactor RHMI-related things they, probably, should be addressed in a separate ticket :thinking: 
What do you think?

Reply:
Ah okay, just wondering overall. I'm good keeping it as it if we want to isolate this to just RHOAM :+1: